packages/quantic/force-app/solutionExamples/main/lwc/exampleInsightPanel/exampleInsightPanel.html

```diff
@@ -85,8 +87,8 @@
           ></c-quantic-category-facet>
         </c-quantic-facet-manager>
 
-        <div class="insight-panel_tabs">
-          <c-quantic-tab-bar>
+        <div class="insight-panel_tabs slds-var-p-horizontal_medium">
+          <c-quantic-tab-bar light-theme>
```

Comment:
what's `light-theme` ?

Reply:
would it not be possible to do something like : `background-color: inherit;` seems a bit overkill to have a public property for a background color, maybe we could eventually set it to be similar to the parent component? is there a situation where we would want the tabbar to be a different color than the parent component?

Reply:
Yes very good proposition 👍 worth trying and applying after confirming it works well with all our use cases.